src/main/java/by/itech/lab/supplier/controller/WarehouseController.java

```diff
@@ -30,4 +37,22 @@
     public WarehouseDto findById(@PathVariable final Long id) {
         return warehouseService.findById(id);
     }
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public WarehouseDto save(@PathVariable(name = "customerId") Long customerId, @Valid @RequestBody WarehouseDto warehouseDto) {
+        return warehouseService.save(customerId, warehouseDto);
+    }
+
+    @PutMapping(URL_ID_PARAMETER)
```

Comment:
If we use @ResponseStatus on some endpoints then let's use it across all application, so to be consistent.

Reply:
when I write this annotation, I concreting response status, another way, i expect status.OK, which is the default status, am I wrong?

Reply:
I agreed, that we shouldnt place annotations which dont change anything